swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -39,90 +39,87 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
     Log.log("\(#function)")
 
     Task {
-      do {
-        // Can be removed after all clients >= 1.4.0
-        try FirezoneId.migrate()
-
-        // The tunnel can come up without the app having been launched first, so
-        // initialize the id here too.
-        let id = try FirezoneId.createIfMissing()
+      // If we don't have a token, we can't continue.
```

Comment:
Most of this diff is:

- Removing the outer `do... catch` in favor of wrapping each `try` as needed. The `adapter.start()` call at the end of this closure is the only one we care to actually error on, upon which we call `completionHandler(error)`.
- Moving the options parsing and token/id loading/saving to helper functions